app/src/main/java/com/example/playlistmaker/root/ui/activity/RootActivity.kt

```diff
@@ -21,13 +22,14 @@ class RootActivity : AppCompatActivity() {
         val bottomNavigationView = findViewById<BottomNavigationView>(R.id.bottomNavigationView)
         bottomNavigationView.setupWithNavController(navController)
         navController.addOnDestinationChangedListener { _, destination, _ ->
-            if (destination.id == R.id.playerActivity) {
+            if (destination.id == R.id.playerActivity || destination.id == R.id.createPlaylistFragment) {
```

Comment:
🍏 Здесь удобно применить when:
```suggestion
            bottomNavigationView.visibility =  when(destination.id) {
            R.id.playerActivity, R.id.createPlaylistFragment -> View.GONE
            else -> View.VISIBLE
```